debugger/src/main/kotlin/org/rust/debugger/runconfig/RsExtraDebugDataLoader.kt

```diff
@@ -0,0 +1,137 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.debugger.runconfig
+
+import com.intellij.notification.NotificationType
+import com.intellij.openapi.diagnostic.Logger
+import com.intellij.openapi.util.io.FileUtil
+import com.intellij.openapi.util.text.StringUtil
+import com.jetbrains.cidr.execution.debugger.CidrDebugProcess
+import com.jetbrains.cidr.execution.debugger.backend.DebuggerCommandException
+import com.jetbrains.cidr.execution.debugger.backend.DebuggerDriver
+import com.jetbrains.cidr.execution.debugger.backend.gdb.GDBDriver
+import com.jetbrains.cidr.execution.debugger.backend.lldb.LLDBDriver
+import org.rust.cargo.project.model.CargoProject
+import org.rust.cargo.project.settings.toolchain
+import org.rust.cargo.runconfig.command.workingDirectory
+import org.rust.debugger.*
+import org.rust.debugger.settings.RsDebuggerSettings
+import org.rust.ide.notifications.showBalloon
+import java.nio.file.InvalidPathException
+
+class RsExtraDebugDataLoader(private val process: CidrDebugProcess, cargoProject: CargoProject?) {
+    private val settings = RsDebuggerSettings.getInstance()
+    private val project = process.project
+    private val toolchain = process.project.toolchain
+    private val threadId = process.currentThreadId
+    private val frameIndex = process.currentFrameIndex
+
+    private val commitHash = cargoProject?.rustcInfo?.version?.commitHash
+
+    private val sysroot: String? by lazy {
+        cargoProject?.workingDirectory?.let { toolchain?.getSysroot(it) }
+    }
+
+    fun load() {
```

Comment:
Maybe `configure` or `setUp` in better?
If final class name contains `Configuration`, `configure` will be preferable